helmcharts/services/alloy/templates/hpa.yaml

```
@@ -0,0 +1,83 @@
+apiVersion: autoscaling/v2
+kind: HorizontalPodAutoscaler
+metadata:
+  name: {{ include "alloy.fullname" . }}
+  namespace: {{ include "alloy.namespace" . }}
+  labels:
+    {{- include "alloy.labels" . | nindent 4 }}
+    app.kubernetes.io/component: availability
+spec:
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: {{ .Values.controller.type  }}
```

YAMLlint (1.38.0)

[warning] too many spaces inside braces

(braces)